packages/modeling/src/primitives/commonChecks.js

```diff
@@ -0,0 +1,19 @@
+// verify that the array has the given dimension, and contains Number values
+const isArray = (dimension, array) => {
+  if (Array.isArray(array) && array.length >= dimension) {
+    return array.every((n) => Number.isFinite(n))
+  }
+  return false
+}
+
+// verify that the value is a Number greater than the constant
+const isGT = (value, constant) => (Number.isFinite(value) && value > constant)
```

Comment:
I have a preference for isGreaterThan over isGT.  Minor thing, but it makes the code more like plain english.  Take it or leave it. :)

Reply:
I actually wrote the changes originally with isGreaterThan() but then I needed isGreaterThanOrEquals() which just seemed moronic for a simple check. It did look up abbreviations for these. :)